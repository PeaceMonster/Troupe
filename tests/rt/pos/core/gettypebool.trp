let val x = true in getType x end