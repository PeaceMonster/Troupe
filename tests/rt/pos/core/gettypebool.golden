2024-04-15T10:41:56.876Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: "boolean"@{}%{}
